node/actors/bft/src/testonly/run.rs

```diff
@@ -359,6 +359,7 @@ async fn twins_receive_loop(
         // Send after taking note of potentially gossipable blocks.
         let send = |msg| {
             if let Some(number) = block_to_gossip(target_port, &msg) {
+                tracing::info!("   ~~~ gossip from={port} to={target_port} number={number}");
```

Comment:
nit: you want these logs to be merged?

Reply:
Yes they were helpful to remind what was happening and that gossiping happens from only a select few.